extensions/amp-story/0.1/animation-presets.js

```diff
@@ -27,22 +27,24 @@ import {
 import {px} from '../../../src/style';
 
 /**
- * A list of panning animation names.
+ * A list of animations that take the whole screen.
  * @private @const {!Array<string>}
  */
-const PANNING_ANIMATION_NAMES = [
+const FULL_SCREEN_ANIMATION_NAMES = [
```

Comment:
nit: slightly prefer "full bleed" over "full screen" here, since technically it's not fullscreen as long as we have the browser bars (and, at some point, we might have an actual "full screen" implementation that is different from this)